tests/integration/targets/aci_interface_policy_link_level/tasks/main.yml

```
@@ -138,6 +151,69 @@
+        when: version.current.0.topSystem.attributes.version is version('5.2', '>=')
+
+      - name: Add Link Level Policy with various speeds (check mode)
+        cisco.aci.aci_interface_policy_link_level:
+          <<: *interface_policy_link_level_present
+          speed: "{{ item }}"
+        check_mode: true
+        loop: "{{ supported_speed }}"
+        register: cm_add_policy_speed
+        loop_control:
+          label: "speed-{{ item }}"
+
+      - name: Add Link Level Policy with various speeds (normal mode)
+        cisco.aci.aci_interface_policy_link_level:
+          <<: *interface_policy_link_level_present
+          speed: "{{ item }}"
+        loop: "{{ supported_speed }}"
+        register: nm_add_policy_speed
+        loop_control:
+          label: "speed-{{ item }}"
+    
+      - name: Verify each speed setting (normal mode)
+        ansible.builtin.assert:
+          that:
+            - "item.1.current.0.fabricHIfPol.attributes.name == 'ansible_test'"
+            - "item.1.current.0.fabricHIfPol.attributes.dn ==  'uni/infra/hintfpol-ansible_test'"
+            - "item.1.current.0.fabricHIfPol.attributes.speed == item.0"
+          quiet: true
+        loop: "{{ supported_speed | zip(nm_add_policy_speed.results) }}"
+        loop_control:
+          label: "speed-{{ item.0 }}"
+
+      - name: Verify each speed setting (check mode)
+        ansible.builtin.assert:
+          that:
+            - "item.1.current.0.fabricHIfPol.attributes.name == 'ansible_test'"
+            - "item.1.current.0.fabricHIfPol.attributes.dn ==  'uni/infra/hintfpol-ansible_test'"
+            - "item.1.current.0.fabricHIfPol.attributes.speed == 'inherit'"
+            - "item.1.proposed.fabricHIfPol.attributes.speed == item.0"
+          quiet: true
+        loop: "{{ supported_speed | zip(cm_add_policy_speed.results) }}"
+        loop_control:
+          label: "speed-{{ item.0 }}"
+    ignore_errors: true
```

What is the purpose of this ignore error?

---

Sorry, left behind and forgotten from testing with the throttling on sandbox. Removed :) .

---

no worries, thanks will run the tests later and will let you know if anything still should change